rpc/documentation/api.md

```diff
@@ -1476,3 +1477,86 @@ The user input can be either a hexadecimal string or a mnemonic word list.
 - `InvalidArgument`: The input is invalid.
 
 **Stability:** Unstable
+
+## `TicketBuyerService`
+
+The `TicketBuyerService` service provides RPC clients with the ability to
+launch the automatic ticket purchaser with given initial configuration
+parameters, and to stop it.  It has no dependencies and is always running.
+
+**Methods:**
+
+- [`StartTicketPurchase`](#startticketpurchase)
+- [`StopTicketPurchase`](#stopticketpurchase)
+
+### Methods
+
+#### `StartTicketPurchase`
+
+The `StartTicketPurchase` method starts an automatic ticket purchaser with the given parameters.
+
+**Request:** `StartTicketPurchaseRequest`
+
+- `bytes passphrase`: The private passphrase to unlock the wallet.
+
+- `uint32 account`: The account number to use for purchasing tickets.
+
+- `int64 balance_to_maintain`: The minimum amount of funds to never dip below when purchasing tickets.
+
+- `int64 max_fee_per_kb`: The maximum ticket fee amount per KB.
+
+- `double max_price_relative`: The scaling factor for setting the maximum ticket price, multiplied by the average price.
+
+- `int64 max_price_absolute`: The maximum absolute ticket price.
+
+- `string voting_address`: The address to delegate voting rights to.
+
+- `string pool_address`: The stake pool address where ticket fees will go to.
+
+- `double pool_fees`: The absolute per ticket fee mandated by the stake pool as a percent.
+
+- `int64 max_per_block`: The maximum tickets per block. Negative number indicates one ticket every n blocks.
+
+**Response:** `StartTicketPurchaseResponse`
+
+**Expected errors:**
+
+- `InvalidArgument`: The private passphrase is incorrect.
+
+- `NotFound`: The account does not exist.
+
+- `InvalidArgument`: An invalid balance to maintain was specified.
+
+- `InvalidArgument`: An invalid maximum ticket fee amount per KB was specified.
+
+- `InvalidArgument`: An invalid maximum ticket price was specified.
+
+- `InvalidArgument`: An invalid voting address was specified.
+
+- `InvalidArgument`: An invalid pool address was specified.
+
+- `InvalidArgument`: Pool address was specified, but pool fees were not.
+
+- `InvalidArgument`: Pool address was not specified, but pool fees were.
+
+- `InvalidArgument`: And invalid pool fees amount was given, either too large or too small.
+
+- `FailedPrecondition`: Ticket buyer is already started.
```

Comment:
needs to document another failed precondition for the wallet not having been loaded yet.